tests/tuner/test_auto_gpu_select.py

```diff
@@ -16,31 +16,11 @@
 import pytest
 import torch
 
-from pytorch_lightning import Trainer
 from pytorch_lightning.tuner.auto_gpu_select import pick_multiple_gpus
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from tests.helpers.runif import RunIf
 
 
-@RunIf(min_gpus=2)
-@pytest.mark.parametrize(
-    ["auto_select_gpus", "gpus", "expected_error"],
-    [(True, 0, MisconfigurationException), (True, -1, None), (False, 0, None), (False, -1, None)],
-)
-def test_trainer_with_gpus_options_combination_at_available_gpus_env(auto_select_gpus, gpus, expected_error):
```

Comment:
Why is this test removed?

I guess the message needs to be updated to work with the new API

cc @kaushikb11

Reply:
Yes! So `auto_select_gpus` doesn't really work with `devices` and `accelerator`.

Reply:
Alternatively, I could wrap the test with `pytest.deprecated_call()`

Reply:
> auto_select_gpus doesn't really work with devices and accelerator. 

Why wouldn't it?

Reply:
I would go with the `deprecated_call` and add a TODO about updating the messaging to mention the new arguments

Reply:
@kaushikb11 That is exactly my question. I’m going to use Paperspace for the next couple of pull requests to help debug faster. Hopefully that speeds up the commits/responses. If anyone knows how to mock multiple gpus, please let me know.

Reply:
Hey @mathemusician. I am curious, what are the advantages of using PaperSpace you don't find elsewhere?

Reply:
> If anyone knows how to mock multiple gpus, please let me know.

You can find multiple examples of exactly that in this file:

`$ grep -iirn "mock" tests/accelerators/test_accelerator_connector.py `

Usually, just `device_count` works, but depends on the test

Reply:
@tchaton I may just be ignorant about the other options, but it was the first thing I saw that I could ssh tunnel into and tweak code while running multiple gpus.

Reply:
I decided to put back the test. I added a TODO for @daniellepintz

Reply:
> grep -iIrn "mock" tests/accelerators/test_accelerator_connector.py

Also that is so cool. I'm like learning so much about `grep` just by doing this.